picard/ui/infodialog.py

```diff
@@ -101,7 +102,7 @@ def __init__(self, obj, parent=None):
         self.ui = Ui_InfoDialog()
         self.display_existing_artwork = False
         if isinstance(obj, File) and isinstance(obj.parent, Track) or \
-                isinstance(obj, Track):
+                isinstance(obj, Track) or isinstance(obj, Album):
```

Comment:
Hmmm, I can merge Track and Album but isinstance(obj, File) has an "and" with isinstance(obj.parent, Track), so I would leave it as:
```
if isinstance(obj, File) and isinstance(obj.parent, Track) or \
        isinstance(obj, (Track, Album)):
```
Deal?